plugins/pagerduty/plugin.spec.yaml

```diff
@@ -77,34 +271,15 @@ actions:
         title: Schedule ID
         description: Schedule ID
         type: string
-        required: false
+        required: true
         example: P9E0DZT
     output:
       users:
         title: Users
         description: List of on-call users
         type: "[]user"
         required: true
-  get_user_by_email:
-    title: Get User by Email
-    description: Get a User by Email
-    input:
-      email:
-        title: Email
-        description: Email
-        type: string
-        required: true
-    output:
-      found:
-        title: Found
-        type: boolean
-        description: True if found
-        required: false
-      user:
-        title: User
-        description: User
-        type: user
-        required: false
+        example: [{"avatar_url": "https://secure.gravatar.com/avatar/c67b6c8ed606dec999bb064ada88cf5c.png?d=mm&r=PG","billed": true,"color": "purple","contact_methods": [{"html_url": null,"id": "P9CU341","self": "https://api.pagerduty.com/users/PYGDZB8/contact_methods/P9CU341","summary": "Default","type": "email_contact_method_reference"}],"coordinated_incidents": [],"description": "","email": "user1@example.com","html_url": "https://api.pagerduty.com/users/PYGDZB8","id": "PYGDZB8","invitation_sent": false,"job_title": "","name": "Test account","notification_rules": [{"html_url": null,"id": "POMMNCV","self": "https://api.pagerduty.com/users/PYGDZB8/notification_rules/POMMNCV","summary": "0 minutes: channel P9CU341","type": "assignment_notification_rule_reference"},{"html_url": null,"id": "PSZNORI","self": "https://api.pagerduty.com/users/PYGDZB8/notification_rules/PSZNORI","summary": "0 minutes: channel P9CU341","type": "assignment_notification_rule_reference"}],"role": "owner","self": "https://api.pagerduty.com/users/PYGDZB8","summary": "test summary","teams": [{"html_url": "https://api.pagerduty.com/teams/PLBLK3G","id": "PLBLK3G","self": "https://api.pagerduty.com/teams/PLBLK3G","summary": "Engineering","type": "team_reference"}],"time_zone": "Europe/London","type": "user"}]
```

Comment:
ensure the examples aren't real data

Reply:
All of these examples were fake data or else taken from the pager duty api docs, but i'll update them to make them easier to look fake